### DISCUSSION TITLE
Funkčnosť Get climate a Set climate

### DISCUSSION THREAD

**Discussion by brano6** [OHF-Voice/intents#1835] [Supported Sentences]
Ahojte mám doma  TRV hlavice ktoré sú v HA ako Climate entity. Ale keď sa spýtam vetu "aká je teplota v obývačke" tak HA vyhodí chybu. Skúšal som aj analyzovať vety vo vývojovom prostredí a dopadlo to nasledovne  viď príloha. Vypadá to tak ako keby tam HA má viacero zhód. Vie mi niekto poradiť vďaka 
![Snímka obrazovky 2023-12-28 164202](https://github.com/home-assistant/intents/assets/148708429/37086a5b-e2a7-46ef-9f12-0e5658cdd5dd)

**Comment by frenck** [comment]
Sorry, we are an English-speaking community. Please try again using the English language.

../Frenck
